src/components/Autocomplete/README.md

```diff
@@ -61,6 +61,14 @@ class AutocompleteExample extends React.Component {
           options={this.state.options}
           selected={this.state.selected}
           onSelect={this.updateSelection}
+          renderEmptyState={
+            <React.Fragment>
+              <Icon source="search" />
+              <div style={{textAlign: 'center'}}>
+                <TextContainer>Could not find any results</TextContainer>
+              </div>
+            </React.Fragment>
+          }
```

Comment:
I can't test this in the style guide because the build is failing.

Reply:
Should we assign the markup to a const first?
```tsx
render() {
    const emptyStateMarkup = (
        ...
     );

     return (
        ...
          renderEmptyState={emptyStateMarkup}
        ...
    )
}
```

Reply:
Same comment for the two examples below

Reply:
TextField, in this example, is not assigned to a variable, so I'm going to leave it as is and clean up the examples in another PR.